### ISSUE TITLE
Backwards compatibility issue with 0.9.3 files

### ISSUE THREAD

**Issue by Cammin** [deepnight/ldtk#633]
The removal of flag type value: `DiscardPreCsvIntGrid` is making 0.9.3 files unable to be deserialized by the json schema for 1.0 schema files.

I understand that it's being deprecated, but maybe this enum value could still exist for just a couple more major updates (into the schema specifically, could effectively be a dead value) to load these old 0.9.3 versions properly until users safely save their project file in 1.0.

Or if it's not going to be added back, then is it more important that all users are _**first**_ expected to load their 0.9.3 LDtk projects and save them in a 1.0 editor first to resolve the flag removal instead?

I understand that it's not a massive issue, but it could certainly trip up some users, I believe.

**Comment by deepnight** [comment]
It should still be possible to enable the old IntGrid data format from Project panel, in Advanced settings 🤔 
Is it broken?

![image](https://user-images.githubusercontent.com/1671695/155883413-cb92bb60-7c72-4562-a72c-0a0ff5f86007.png)

My recommendation from API perspective is to make sure the file version is at least 1.0.0 as soon as the app is officially released. LDtk takes care of all retro-compatibility up to 0.2.x versions, so API should just always focus on latest versions, imho.

**Comment by Cammin** [comment]
I can give a more specific scenario from my own perspective.

For my importer package: Once LDtk version 1.0 releases, then I will publish a package update for the new 1.0 content support. After users update the package, any user who did not yet save their 0.9.3 project in 1.0 to correct this Flag value change, will get a deserialization bug from the quicktype `FlagConverter` displaying that the `DiscardPreCsvIntGrid` Flag value cannot be unmarshalled.
At least, it's what I'm experiencing with all of the sample files with my tests.
I have since worked around the issue by modifying the quicktype converter class for Flag by inserting the since-removed flag value: 
![image](https://user-images.githubusercontent.com/55564581/155892307-08f088ee-f6fc-4f23-a950-3891e5f63083.png)
With these workarounds, it means I need to make sure that I'm not overwriting these specific quicktype-generated files. So it would be maintenance work to make sure it's working properly for every new schema regeneration.

In 0.9.3, the Flag values are: 
```
public enum Flag { DiscardPreCsvIntGrid, IgnoreBackupSuggest };
```
In 1.0, the Flag values are:
```
public enum Flag { ExportPreCsvIntGridFormat, IgnoreBackupSuggest, MultiWorlds, PrependIndexToLevelFileNames, UseMultilinesType };
```

The `DiscardPreCsvIntGrid` value was removed between the versions.

Of course, if this was the intention that users are always expected to re-save their projects before attempting to import, then this isn't an issue. But I want your confirmation if that's the case 🙂
